.github/workflows/build.yml

```diff
@@ -46,6 +46,7 @@ jobs:
   
   analysis:
     name: Analyze with Sonar Cloud
+    if: github.repository == 'sap/cap-operator-lifecycle'
```

Comment:
Do we really need to do this?
Couldn't  we just remove these parts in the forks - a lot of these parts may not run anyway and need more ifs.. like here.

Reply:
Most of the other workflows (so far) are running well - including package publishing (helm / container) in forks. As far as I tried, this is the only one which is problematic. I would prefer to keep it. I can maybe change the condition to a `github.repository_owner == 'sap'` instead of `repository`.

Reply:
I don't mean the other workflows, that would be good.
However, IMO we shouldn't be coding for forks in this repo w.r.t adding such if checks (sonar, other repos etc..). Let the actions fail (or be removed where not needed) when running e.g. in forks.